### DISCUSSION TITLE
What is the new method to replace the deprecated method volcanoClient.queues().createOrReplace(queue)?

### DISCUSSION THREAD

**Discussion by hzp01** [fabric8io/kubernetes-client#5652] [Q&A] [answered]
What is the new method to replace the following deprecated methods： 
1、volcanoClient.queues().createOrReplace(queue)
2、volcanoClient.queues().delete(queue)
3、volcanoClient.podGroups().createOrReplace(podGroup)
4、volcanoClient.podGroups().delete(podGroup)

**Comment by rohanKanojia** [comment, marked as answer]
> 1、volcanoClient.queues().createOrReplace(queue)

volcanoClient.queues().resource(queue).serverSideApply();

> 2、volcanoClient.queues().delete(queue)

volcanoClient.queues().resource(queue).delete();

> 3、volcanoClient.podGroups().createOrReplace(podGroup)

volcanoClient.podGroups().resource(podGroup).serverSideApply();

> 4、volcanoClient.podGroups().delete(podGroup)

volcanoClient.podGroups().resource(podGroup).delete();

For more information regarding common operations, you can refer to [CHEATSHEET](https://github.com/fabric8io/kubernetes-client/blob/main/doc/CHEATSHEET.md#kubernetes-client-dsl-usage).
